src/components/snake.css

```diff
@@ -0,0 +1,6 @@
+.snake{
```

Comment:
Let's add a space before open curly bracket (`{`), please 🙏

Reply:
why you need to add a space before open curly bracket? 🤔

Reply:
If we follow linter rules for CSS like [`stylelint`](https://stylelint.io), you will see that spaces are good options for writing css rules. You can paste your code and try it [here](https://stylelint.io/demo/).

Reply:
Actually, I think we should add a linter for CSS 🤔